stl/inc/deque

```diff
@@ -1610,13 +1610,13 @@ _NODISCARD bool operator>=(const deque<_Ty, _Alloc>& _Left, const deque<_Ty, _Al
 
 #if _HAS_CXX20
 template <class _Ty, class _Alloc, class _Uty>
-void erase(deque<_Ty, _Alloc>& _Cont, const _Uty& _Val) {
-    _Erase_remove(_Cont, _Val);
+typename deque<_Ty>::size_type erase(deque<_Ty, _Alloc>& _Cont, const _Uty& _Val) {

```

Comment:
That's not the correct type since size_type comes from _Alloc

Reply:
Whoops!